assets/styles/layout/_forms.scss

```diff
@@ -151,6 +151,7 @@ input[type=select-one] {
   display: block;
   padding: 1rem .5rem;
   width: 100%;
+  text-indent: .1rem !important;
```

Comment:
What is this for?

Reply:
I noticed that the text entry was still being cutoff a little without it:

before

![image](https://github.com/MbinOrg/mbin/assets/35878315/81eb439c-cfa1-4e80-96b8-d9ec9506b440)

after

![image](https://github.com/MbinOrg/mbin/assets/35878315/faeef72c-6aeb-4b85-b070-d91d79087ccc)

this autocomplete thing pulls in it's own stylesheet that seems to really be messing things up with our css...

![image](https://github.com/MbinOrg/mbin/assets/35878315/b22dbc6a-aaea-4be8-a13b-d0c2c879616e)

Reply:
It does seem to effect all input boxes on the site though, was that expected, or would only the auto complete ones need it?

Reply:
that badges form isn't autocomplete enabled and it still needed it if you can tell by the image...i know, it's super subtle. i guess my assumption about this autocomplete css import causing problems isn't totally 100% accurate (although, maybe it is?) i dunno, seems like we have a lot of suboptimal stuff going on.

Reply:
Ah, I see, sorry, I got confused because I was like, you could just remove `!important` and remove the `0 !important` where it is but then I realized as you said it's coming from symfony so they don't really give you the option of doing that